sdk/servicebus/service-bus/test/internal/unit/utils.spec.ts

```diff
@@ -341,7 +337,7 @@ describe("utils", () => {
       };
 
       extractSpanContextFromServiceBusMessage(receivedMessage);
-      assert.isTrue(tracingClientSpy.calledWith(traceparent));
+      assert.equal(tracingClientSpy.mock.calls[0][0], traceparent);
```

Comment:
Could we not use `expect(tracingClientSpy).toHaveBeenCalledWith(traceparent)`?  If there are multiple parameters we could also say `expect(tracingClientSpy).toHaveBeenCalledWith(traceparent, expect.anything())`

Reply:
updated.